sample_app/app/controllers/users_controller.rb

```diff
@@ -11,16 +19,57 @@ def new
   def create
     @user = User.new(user_params)
     if @user.save
+      log_in @user
       flash[:success] = "Welcome to the Sample App!"
       redirect_to @user
     else
       render 'new'
     end
   end
   
+  def edit
```

Comment:
how do we know this is protected?  Need to read all the filters that may apply to this method.  And they're not necessarily all in this file!  How complicated things can get...